packages/facility-server/app/routes/apiv1/invoice/getPotentialInvoiceItems.js

```diff
@@ -36,7 +36,6 @@ export const getPotentialInvoiceItems = async (db, invoiceId, imagingTypes) => {
 	select
 		ip.id,
 		ip.name,
-		ip.price,
     ip.discountable
```

Comment:
potential invoice items will be deleted when automatic invoicing is complete so this change is to support the rest of the changes but won't matter soon.